docs/common/istio-ambient-update.adoc

```diff
@@ -0,0 +1,653 @@
+
+. Update the IstioCNI resource to the same version as the control plane:
++
+[source,bash,subs="attributes+",name="ambient-inplace-update-strategy"]
+----
+$ kubectl patch istiocni -n istio-cni default --type='merge' -p '{"spec":{"version":"v1.26.4"}}'
+----
++
+Wait for the IstioCNI resource to become ready:
++
+[source,bash,subs="attributes+",name="ambient-inplace-update-strategy"]
+----
+$ kubectl wait --for=condition=Ready istiocnis/default --timeout=5m
+----
+
+. Update the ZTunnel resource to the same version as the control plane:
++
+[source,bash,subs="attributes+",name="ambient-inplace-update-strategy"]
+----
+$ kubectl patch ztunnel -n ztunnel default --type='merge' -p '{"spec":{"version":"v1.26.4"}}'
+----
++
+Wait for the ZTunnel resource to become ready:
++
+[source,bash,subs="attributes+",name="ambient-inplace-update-strategy"]
+----
+$ kubectl wait --for=condition=Ready ztunnel/default --timeout=10m
+----
++
+*Note:* The ZTunnel DaemonSet update may take several minutes as pods are updated node-by-node to minimize disruption.
+
+For detailed information about updating IstioCNI, see link:../../README.md#updating-istiocni[IstioCNI Update Procedure in README]. For ZTunnel update procedures, refer to section "Common Update Procedures for Ambient Components".
+
+[[recommendations-for-inplace-strategy-in-ambient-mode]]
+=== Recommendations for InPlace strategy in Ambient Mode
+
+* **High Availability:** Configure the istiod deployment with multiple replicas to help reduce disruptions during updates. Note that even with HA mode, some brief traffic disruption may still occur during control plane upgrades. See the link:../general/istiod-ha.adoc#running-istiod-in-ha-mode[Istiod HA guide] for more information.
+* **ZTunnel Updates:** The ZTunnel DaemonSet uses a RollingUpdate strategy by default, which updates pods one node at a time. Monitor the rollout to ensure it completes successfully.
+* **Maintenance Window:** While ambient mode is designed to minimize disruption, it's recommended to perform upgrades during a maintenance window.
+* **Testing:** Always test the upgrade process in a non-production environment first.
+
+'''
+
+[[updating-with-revisionbased-strategy]]
+== Updating with RevisionBased Strategy
+
+For general information about the RevisionBased strategy, see link:../update-strategy/update-strategy.adoc#revisionbased[RevisionBased].
+
+WARNING: **RevisionBased updates are not fully supported in ambient mode and require significant manual intervention.** This strategy has the following limitations when used with ambient mode:
+
+* **Single ZTunnel Instance:** Only one ztunnel instance can run in the cluster at any time, which limits the canary upgrade capabilities typically available with RevisionBased updates.
+* **Manual CRD Synchronization:** You must manually synchronize CRDs between Istio control plane revisions and the ZTunnel instance, primarily managing the revision name configuration.
```

Comment:
When you say, synchronize, are you referring to the values that are part of the ztunnel resource/CR and the Istio resource/CR? Or are you talking about CRD itself? If its the former, please rephrase it.